Cargo.toml

```diff
@@ -17,19 +17,18 @@ mio-extras = "~2.0.5"
 net2 = "~0.2.30"
 quick-error = "~1.2.0"
 rand = "~0.3.16"
-rust_sodium = "~0.5.0"
+rust_sodium = "~0.10.2"
 serde = "~1.0.11"
 serde_derive = "~1.0.11"
-unwrap = "~1.1.0"
+unwrap = "~1.2.1"
 
 [dependencies.socket-collection]
-version = "~0.3.0"
-#git = "https://github.com/maidsafe/socket-collection"
-#branch = "community-release-1"
+version = "~0.4.0"
+git = "https://github.com/ekutt/socket-collection"
```

Comment:
We also need to update some dependencies in maidsafe/socket-collection repo. Maybe we need to fork it and merge the changes into our repo.

Reply:
forked into /nblockchain/ org thanks, you can update the PR

Reply:
> forked into /nblockchain/ org thanks, you can update the PR

Thank you! Created a pull request for that also.

Reply:
What pull request? I thought this had to just change in this PR, I don't understand

Reply:
> What pull request? I thought this had to just change in this PR, I don't understand
There are two dependencies (maidsafe_utilities, safe_crypto) that need to be updated in socket-collection repo also, otherwise it doesn't build. After it gets merged into nblockchain/socket-collection, I will update the socket-collection dependency here to reference to nblockchain/socket-collection repo:
https://github.com/nblockchain/socket-collection/pull/1